From 013efeaa11ef7894123fae2558cb997e80ddb296 Mon Sep 17 00:00:00 2001
From: Charles Short <charles.short@windriver.com>
Date: Mon, 29 Nov 2021 16:05:46 -0500
Subject: [PATCH 1/2] Add wheels support

Add support for building python3 wheel for python-barbicanclient

Signed-off-by: Charles Short <charles.short@windriver.com>

[resave as unified diff, add git log]
Signed-off-by: Michel Thebeau <Michel.Thebeau@windriver.com>
---
 debian/changelog |  8 ++++++++
 debian/control   | 13 +++++++++++++
 debian/rules     |  2 +-
 3 files changed, 22 insertions(+), 1 deletion(-)

diff --git a/debian/changelog b/debian/changelog
index 534a14c..e96552c 100644
--- a/debian/changelog
+++ b/debian/changelog
@@ -1,3 +1,11 @@
+python-barbicanclient (7.1.0-3.1) unstable; urgency=medium
+
+  * Non-maintainer upload.
+  * debian/control, debian/rules: Add wheels support. 
+  * Rebase onto 7.1.0-3; Michel Thebeau
+
+ -- Chuck Short <charles.short@windriver.com>  Sun, 03 Oct 2021 18:30:48 +0000
+
 python-barbicanclient (7.1.0-3) unstable; urgency=medium
 
   * Uploading to unstable.
diff --git a/debian/control b/debian/control
index 3b0a473..66dff50 100644
--- a/debian/control
+++ b/debian/control
@@ -54,3 +54,16 @@ Description: OpenStack Key Management API client - Python 3.x
  command-line script (barbican).
  .
  This package contains the Python 3.x module.
+
+Package: python3-barbicanclient-wheel
+Architecture: all
+Depends:
+ python3-wheels,
+ ${misc:Depends},
+ ${python3:Depends},
+Description: OpenStack Key Management API client - Python 3.x
+ This is a client for the Barbican Key Management API. This package includes a
+ Python library for accessing the API (the barbicanclient module), and a
+ command-line script (barbican).
+ .
+ This package contains the Python 3.x wheel. 
diff --git a/debian/rules b/debian/rules
index 396d533..4ea090c 100755
--- a/debian/rules
+++ b/debian/rules
@@ -14,7 +14,7 @@ override_dh_auto_build:
 	echo "Do nothing..."
 
 override_dh_auto_install:
-	pkgos-dh_auto_install --no-py2
+	pkgos-dh_auto_install --no-py2 --wheel
 
 override_dh_auto_test:
 ifeq (,$(findstring nocheck, $(DEB_BUILD_OPTIONS)))
-- 
2.34.1

